### PR TITLE
fix(sheet): add from & to range in moverange & clone the cellvalue

### PR DESCRIPTION
**Pull request by VicKun4937**
-  add param {fromRange, toRange} in moveRange mutation for pivot table 
- clone the cell value to resolve bugs

### CODE REVIEW

**Review by wzhudev — APPROVED** [review]
LGTM

File: packages/sheets/src/commands/mutations/move-range.mutation.ts [hunk 1]
```diff
@@ -14,11 +14,13 @@
  * limitations under the License.
  */
 
-import type { ICellData, IMutation, IObjectMatrixPrimitiveType, Nullable, Workbook } from '@univerjs/core';
+import type { ICellData, IMutation, IObjectMatrixPrimitiveType, IRange, Nullable, Workbook } from '@univerjs/core';
 import { CommandType, IUniverInstanceService, ObjectMatrix, UniverInstanceType } from '@univerjs/core';
 
 export interface IMoveRangeMutationParams {
     unitId: string;
+    fromRange: IRange;
```
Comment (Gggpound) [inline packages/sheets/src/commands/mutations/move-range.mutation.ts:22]: 这个参数会落盘的，无意义啊.

Comment (VicKun4937) [reply packages/sheets/src/commands/mutations/move-range.mutation.ts:22]: @yuhongz @wzhudev  透视表这里只关心fromrange的范围，有其他的好的解法吗

**Comment by VicKun4937** [comment]
@wzhudev  please review again: 
the add changes files:
packages/sheets/src/services/ref-range/util.ts
packages/sheets-ui/src/controllers/render-controllers/sheet.render-controller.ts
 it seems last editor by you

**Pull request merged**